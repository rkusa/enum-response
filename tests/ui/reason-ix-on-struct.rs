#[macro_use]
extern crate api_error_derive;

#[derive(ErrorStatus)]
enum Error {
    #[response(reason_field = 0)]
    Struct { s: String }
}

