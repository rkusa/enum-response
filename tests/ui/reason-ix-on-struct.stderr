error: proc-macro derive panicked
 --> $DIR/reason-ix-on-struct.rs:4:10
  |
4 | #[derive(ErrorStatus)]
  |          ^^^^^^^^^^^
  |
  = help: message: reason index only works for tuple variants

